src/frontends/common/src/exception.cpp

```diff
@@ -8,33 +8,33 @@ void ov::frontend::GeneralFailure::create(const CheckLocInfo& check_loc_info,
                                           const std::string& context_info,
                                           const std::string& explanation) {
     throw ov::frontend::GeneralFailure(
-        make_what(check_loc_info, "FrontEnd API failed with GeneralFailure: " + context_info, explanation));
+        make_what(check_loc_info, "FrontEnd API failed with GeneralFailure" + context_info, explanation));
 }
 
 void ov::frontend::InitializationFailure::create(const CheckLocInfo& check_loc_info,
                                                  const std::string& context_info,
                                                  const std::string& explanation) {
     throw ov::frontend::InitializationFailure(
-        make_what(check_loc_info, "FrontEnd API failed with InitializationFailure: " + context_info, explanation));
+        make_what(check_loc_info, "FrontEnd API failed with InitializationFailure" + context_info, explanation));
 }
 
 void ov::frontend::OpValidationFailure::create(const CheckLocInfo& check_loc_info,
                                                const std::string& context_info,
                                                const std::string& explanation) {
     throw ov::frontend::OpValidationFailure(
-        make_what(check_loc_info, "FrontEnd API failed with OpValidationFailure: " + context_info, explanation));
+        make_what(check_loc_info, "FrontEnd API failed with OpValidationFailure" + context_info, explanation));
 }
 
 void ov::frontend::OpConversionFailure::create(const CheckLocInfo& check_loc_info,
                                                const std::string& context_info,
                                                const std::string& explanation) {
     throw ov::frontend::OpConversionFailure(
-        make_what(check_loc_info, "FrontEnd API failed with OpConversionFailure: " + context_info, explanation));
```

Comment:
I am not sure that we should remove `:`
Logically, below it is going explanation of some failure

Reply:
Yes we should. ":" are added in `make_what` function, if we add them here we have double `: :`